spec/features/article_searching_spec.rb

```diff
@@ -41,4 +41,38 @@
 
     expect(current_url).to match(%r{/article\?.*&q=Kittens})
   end
+  describe 'breadcrumbs', js:true do
+   scenario 'start over button returns users to articles home page' do
+     skip('need to stub out Eds::SearchService') if ENV['CI']
+     visit article_home_path
```

Comment:
@jkeck this pattern is starting to get a little redundant. Maybe I could DRY this out and add a helper?